templates/zerver/help/make-an-announcement.md

```diff
@@ -15,12 +15,13 @@ Announcements on Zulip are sent using the `#announce` stream.
 4. Enter the topic for your announcement in the **Topic** field.
 
     !!! tip ""
-        Your topic name can be no longer than 52 characters.
+        Your topic name cannot be longer than 52 characters.
 
 5. Enter your announcement in the box labeled
- **Compose your message here...** or messaging box.
+ **Compose your message here...** (also known as the messaging box).
```

Comment:
I think it's compose box, according to the glossary - https://github.com/zulip/zulip/blob/e66abbebf99c5a8f2875c6be55bd49acd700aba8/templates/zerver/help/zulip-glossary.md#compose-box